api/tools/src/main/java/org/pmiops/workbench/tools/CommandLineToolConfig.java

```diff
@@ -0,0 +1,76 @@
+package org.pmiops.workbench.tools;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.http.HttpTransport;
+import com.google.api.client.http.apache.ApacheHttpTransport;
+import com.google.gson.Gson;
+import org.pmiops.workbench.config.WorkbenchConfig;
+import org.pmiops.workbench.db.dao.ConfigDao;
+import org.pmiops.workbench.db.model.Config;
+import org.springframework.context.annotation.*;
```

Comment:
Should avoid `*` imports. I think you can switch this in Intellij (another thing to cover with the linter choice).